controller/userWebController.js

```diff
@@ -1946,6 +1946,67 @@ module.exports = {
                 db_connection.release();
             }
         }
+    ], 
+
+    editManagerProfileById: [
+        /*
+        JSON
+        {
+            "managerId" : "<managerId>" INTEGER,
+            "managerEmail" : "<managerEmail>",
+            "managerPassword" : "<managerPassword>",
```

Comment:
The `password` update will happen thru forget password and this. Insecure way! Also, email cannot be updated by the manager.